docs/source/en/tasks/object_detection.md

```diff
@@ -46,11 +46,11 @@ The task illustrated in this tutorial is supported by the following model archit
 Before you begin, make sure you have all the necessary libraries installed:
 
 ```bash
-pip install -q datasets transformers evaluate timm albumentations
+pip install -q datasets transformers accelerate evaluate albumentations
```

Comment:
The DETR models support the "no_timm" revision, which has been asked by several users. It allows to run the models with the AutoBackbone class rather than the Timm backbone.

Reply:
I understand the need for this in general, but I'm just wondering why change for this example given we already depend on third party libraries?

Reply:
Simply because I prefer simplicity, so if users can run this tutorial without one additional dependency, I guess that's great for them